client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss

```diff
@@ -34,10 +34,25 @@ $white: var( --color-surface );
 		transform-origin: 0 0;
 	}
 
-	.progress-bar {
+	&:not( .newsletters ):not( .link-in-bio ) .progress-bar {
 		display: none;
```

Comment:
I think we could entirely remove this selector. I guess we should always show the progress bar and it is by default visible.

Reply:
Would that add the css also to other flows, where we do not want the progress bar?